src/Symfony/Component/Validator/Constraints/Iban.php

```diff
@@ -23,18 +23,18 @@
  */
 class Iban extends Constraint
 {
-    const TOO_SHORT_ERROR = 1;
-    const INVALID_COUNTRY_CODE_ERROR = 2;
-    const INVALID_CHARACTERS_ERROR = 3;
-    const INVALID_CASE_ERROR = 4;
-    const CHECKSUM_FAILED_ERROR = 5;
+    const INVALID_COUNTRY_CODE_ERROR = 1;
+    const INVALID_CHARACTERS_ERROR = 2;
+    const CHECKSUM_FAILED_ERROR = 3;
+    const INVALID_FORMAT_ERROR = 4;
+    const NOT_SUPPORTED_COUNTRY_CODE_ERROR = 5;
```

Comment:
You should revert the value changes as it would be a BC break. You should also keep the `TOO_SHORT_ERROR` to avoid BC breaks, and deprecate it so that we can remove it in 3.0.